logux-vue-todomvc/README.md

```diff
@@ -0,0 +1,8 @@
+# logux-vue-todomvc
```

Comment:
Here is structure that we should keep

https://github.com/logux/vuex/issues/24#issuecomment-623171675